torch/optim/__init__.py

```diff
@@ -37,3 +37,6 @@
 del optimizer  # type: ignore[name-defined] # noqa: F821
 del nadam  # type: ignore[name-defined] # noqa: F821
 del lbfgs  # type: ignore[name-defined] # noqa: F821
+
+
+import torch.optim._multi_tensor
```

Comment:
This was removed in https://github.com/pytorch/pytorch/pull/128875

What is the reason for adding it back?

Reply:
It caused some test failures. The specific order in which imports were done caused some tests to not be able to resolve "torch.optim._multi_tensor".